macros/edr/system/system_utils/empty_table.sql

```diff
@@ -1,5 +1,5 @@
 {% macro empty_elementary_test_results() %}
-    {{ elementary.empty_table([('id','long_string'),('data_issue_id','string'),('test_execution_id','long_string'),('test_unique_id','long_string'),('model_unique_id','long_string'),('detected_at','timestamp'),('database_name','string'),('schema_name','string'),('table_name','string'),('column_name','string'),('test_type','string'),('test_sub_type','string'),('test_results_description','long_string'),('owners','string'),('tags','string'),('test_results_query','long_string'),('other','string'),('test_name','long_string'),('test_params','long_string'),('severity','string'),('status','string')]) }}
+    {{ elementary.empty_table([('id','long_string'),('data_issue_id','string'),('test_execution_id','long_string'),('test_unique_id','long_string'),('model_unique_id','long_string'),('detected_at','timestamp'),('database_name','string'),('schema_name','string'),('table_name','string'),('column_name','string'),('test_type','string'),('test_sub_type','string'),('test_results_description','long_string'),('owners','string'),('tags','string'),('test_results_query','long_string'),('other','string'),('test_name','long_string'),('test_params','long_string'),('severity','string'),('status','string'),('test_node_name', 'string'),('test_alias', 'string')]) }}
```

Comment:
I think that having both test_node_name and test_name is confusing - we need to either change test_name to be test_short_name and then test_name will be the long but then we will need to think how to support backwards. That's the ideal solution (we can do it by checking if the test_short_name exists use it and if not use the test_name). If that's a big hassle we can maybe rename test_node_name to test_long_name

Reply:
I am really against giving names that doesn't match anything in the graph to our fields (like test long name).
It is much more confusing and you can't understand why it is called like that.
Test node name is pretty obvious that is the name of the test node (node.name) IMO.

We can start and make changes in the existing tables and call it short_name and name, but that might cost us more (but if that is the right way then that is the right way).